custom/icds/tasks/sms.py

```diff
@@ -68,11 +69,13 @@ def send_monthly_sms_report():
 
 
 @task
-def send_custom_sms_report(start_date, end_date, email):
+def send_custom_sms_report(start_date: str, end_date: str, email, domain):
```

Comment:
nit: i am not sure what the standard is, why not mention that for all args?

Reply:
Yep. That will make things more consistent